### PR TITLE
Now methods async with cancellation token.

### PR DESCRIPTION
**Pull request by VideoRab**

### CODE REVIEW

File: HumanResourceMachine/Controllers/HumanController.cs [hunk 1]
```diff
@@ -22,41 +22,41 @@ public HumanController(IHumanService service, IMapper mapper)
...
-            var humans = _service.GetAllHumans();
+            var humans = await _service.GetAllHumans(token);
             var result = _mapper.Map<IEnumerable<HumanViewModel>>(humans);
 
             return result;
         }
 
         [HttpGet("{id}")]
-        public HumanViewModel GetHumanById([FromRoute] int id)
+        public async Task<HumanViewModel> GetHumanById([FromRoute] int id)
```
Comment (GodlyCat) [inline HumanResourceMachine/Controllers/HumanController.cs:34]: where ct ?

File: HRM.DAL/Interfaces/IHumanRepository.cs [hunk 2]
```diff
@@ -4,10 +4,10 @@ namespace HRM.DAL.Interfaces.Repository
 {
     public interface IHumanRepository
     {
-        IEnumerable<HumanEntity> GetAllHumans();
-        HumanEntity GetHumanById(int id);
-        void AddHuman(HumanEntity human);
-        void UpdateHuman(HumanEntity human);
-        void DeleteHumanById(int id);
+        Task<IEnumerable<HumanEntity>> GetAllHumans(CancellationToken token);
+        Task<HumanEntity> GetHumanById(int id);
```
Comment (GodlyCat) [inline HRM.DAL/Interfaces/IHumanRepository.cs:8]: where ct

File: HRM.BLL/Interfaces/IHumanService.cs [hunk 3]
```diff
@@ -4,10 +4,10 @@ namespace HRM.BLL.Interfaces
 {
     public interface IHumanService
     {
-        IEnumerable<Human> GetAllHumans();
-        Human GetHumanById(int id);
-        void AddHuman(Human human);
-        void UpdateHuman(Human human);
-        void DeleteHumanById(int id);
+        Task<IEnumerable<Human>> GetAllHumans(CancellationToken token);
+        Task<Human> GetHumanById(int id);
```
Comment (GodlyCat) [inline HRM.BLL/Interfaces/IHumanService.cs:8]: Where ct ?

**Review by GodlyCat — CHANGES_REQUESTED** [review]

File: HRM.DAL/Interfaces/IHumanRepository.cs (hunk 2 above)
Comment (VideoRab) [reply HRM.DAL/Interfaces/IHumanRepository.cs]: Added.

File: HRM.BLL/Interfaces/IHumanService.cs (hunk 3 above)
Comment (VideoRab) [reply HRM.BLL/Interfaces/IHumanService.cs]: Added.

**Review by GodlyCat — APPROVED** [review]

**Review by hromau — APPROVED** [review]

**Pull request merged**